packages/simple-form/README.md

```diff
@@ -11,13 +11,16 @@ React-registry name: `simple-form`
 |------------------------|:---------:|-------------
 | `form`                 | x         | Full form definition 
 | `model`                | x         | Model of the fields that are present in the form
+| `noButtons`            |           | Whether submit and cancel buttons should be shown and the form is controlled from outside. (Default: false)
```

Comment:
(usually there is no period at the end of the description ("outside.") -> formData desc. as well)